zubhub_frontend/zubhub/src/views/edit_team/EditTeam.jsx

```diff
@@ -308,47 +257,63 @@ function EditTeam(props) {
                       fullWidth
                       customButtonStyle
                     >
-                      {t('editProfile.inputs.submit')}
+                      {'Save Changes'}
                     </CustomButton>
                   </Grid>
-                  <Grid item xs={12} sm={12} md={12}>
+                  </Grid>
+                  <Grid container spacing={3}>
+                  <Grid item xs={12}>
+                    <Link to={`/teams/${groupname}`} className={classes.textDecorationNone}>
+                      <CustomButton
+                        variant="outlined"
+                        size="large"
+                        secondaryButtonStyle
+                        customButtonStyle
+                        fullWidth
+                      >
+                        {'Discard Changes'}
```

Comment:
Consider translating this

Reply:
Actually I was waiting for another PR to be merged to translate the texts. That is merged, so I am working on translating these as well, thank you! 🙂